database/migrations/2025_03_10_041236_add_profile_id_to_users_table.php

```
@@ -0,0 +1,39 @@
+return new class () extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('users', function (Blueprint $table) {
+            // Obtém o ID do perfil padrão 'member'
+            $memberProfileId = DB::table('profiles')->where('type', 'member')->value('id');
+
+            $table->foreignId('profile_id')
```

PHPStan (2.0.3)

Call to an undefined method Illuminate\Database\Schema\ForeignKeyDefinition::default().


(method.notFound)